content/en/docs/setup/production-environment/container-runtimes.md

```diff
@@ -465,120 +435,92 @@ Use the following commands to install Docker on your system:
 {{< tabs name="tab-cri-docker-installation" >}}
 {{% tab name="Ubuntu 16.04+" %}}
 
-```shell
-# (Install Docker CE)
-## Set up the repository:
-### Install packages to allow apt to use a repository over HTTPS
-sudo apt-get update && sudo apt-get install -y \
-  apt-transport-https ca-certificates curl software-properties-common gnupg2
-```
+1. Setup the [Docker Engine repository for Ubuntu](https://docs.docker.com/engine/install/ubuntu/#set-up-the-repository)
 
-```shell
-# Add Docker's official GPG key:
-curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key --keyring /etc/apt/trusted.gpg.d/docker.gpg add -
-```
+2. Install Docker CE:
 
-```shell
-# Add the Docker apt repository:
-sudo add-apt-repository \
-  "deb [arch=amd64] https://download.docker.com/linux/ubuntu \
-  $(lsb_release -cs) \
-  stable"
-```
+   ```shell
+   sudo apt-get update && sudo apt-get install -y \
+     containerd.io=1.2.13-2 \
+     docker-ce=5:19.03.11~3-0~ubuntu-$(lsb_release -cs) \
+     docker-ce-cli=5:19.03.11~3-0~ubuntu-$(lsb_release -cs)
+   ```
 
-```shell
-# Install Docker CE
-sudo apt-get update && sudo apt-get install -y \
-  containerd.io=1.2.13-2 \
-  docker-ce=5:19.03.11~3-0~ubuntu-$(lsb_release -cs) \
-  docker-ce-cli=5:19.03.11~3-0~ubuntu-$(lsb_release -cs)
-```
+3. Set up the Docker daemon:
```

Comment:
i think we should remove the tabs here and just link to this page and tell the users to follow the steps that the docker docs already have:
https://docs.docker.com/engine/install/#server

the only follow up step we want to have here is about setuping the Docker daemon.

Reply:
Hmm, the we seem to be installing specific versions here that are required - Earlier we say:

> On each of your nodes, install Docker CE. The Kubernetes release notes list which versions of Docker are compatible with that version of Kubernetes.

Running the install steps on docker would install the latest versions which might break Kubernetes? Though I'm not sure. The Redhat-based configuration steps differ from the Debian ones have slightly different configuration
one has an additional:
```json
{
  "storage-opts": [
      "overlay2.override_kernel_check=true"
    ]
}    
```
So we might still need tabs?

Reply:
> Running the install steps on docker would install the latest versions which might break Kubernetes?

this hasn't been the case for a long time. i think we should recommend 20.10 for Linux*, which is validated in kops e2e test jobs and becoming the "latest validated version" in k8s 1.21.

>  The Redhat-based configuration steps differ from the Debian ones have slightly different configuration
one has an additional:

if needed, two tabs for "Debian-based distributions" and "Red hat-based distributions" make sense.